frontend/src/App/App.css

```diff
@@ -1,3 +1,4 @@
 .App {
   text-align: center;
 }
+
```

Comment:
Remove

Reply:
you mean the extra line or .App?

Reply:
Just the extra line to keep the PR in scope